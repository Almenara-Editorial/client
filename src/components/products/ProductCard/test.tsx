import { render, screen } from '@testing-library/react'

import { ProductCard } from '.'

describe('<ProductCard />', () => {
  it('should render the heading', () => {
    render(<ProductCard />)

    expect(screen.getByRole('heading', { name: /ProductCard/i })).toBeInTheDocument()
  })
})
