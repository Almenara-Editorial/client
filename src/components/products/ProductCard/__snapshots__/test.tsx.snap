// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductCard /> should render the heading 1`] = `
<div
  class=""
>
  <h1>
    ProductCard
  </h1>
</div>
`
